src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/params.py

```diff
@@ -19,6 +19,11 @@
     c.register('condition', ('--condition',),
                type=json.loads,
                help='JSON encoded condition configuration. Use @{file} to load from a file.')
+    c.register('actions', ('--actions',),
+               type=json.loads,
```

Comment:
Cool. Using `get_json_object` instead to take care for snake_case in-addition to `shell_sage_json_parse`